src/App.js

```diff
@@ -1,19 +1,41 @@
 import React from 'react';
 import styled from 'styled-components/macro'
-import DashboardPage from './dashboardPage/DashboardPage'
+import Navbar from './navigation/Navigation'
+import {
+  BrowserRouter as Router,
+Route, 
+Switch,
+} from "react-router-dom"
+import CreatePage from './create/CreatePage'
+import SearchPage from './search/SearchPage'
+import DashboardPage from './dashboardPage/DashboardPage';
```

Comment:
Inconsistent naming: You could either add "page" to the `create`/`search` folder names, or remove it from the `dashboardPage` folder name